common/djangoapps/student/views.py

```diff
@@ -1740,6 +1747,10 @@ def create_account_with_params(request, params):
     if third_party_auth.is_enabled() and pipeline.running(request):
         running_pipeline = pipeline.get(request)
         third_party_provider = provider.Registry.get_from_pipeline(running_pipeline)
+        # EnterpriseCustomers (part of the `enterprise` app) tied to a third-party
```

Comment:
Okay, then maybe we just need to rewrite the comment a bit -- how about something like "Store the 'data_sharing_consent' field value for use by downstream pipeline elements."